waku/waku_enr/sharding.nim

```diff
@@ -27,8 +27,8 @@ const
 
 type
   RelayShards* = object
-    clusterId: uint16
-    shardIds: seq[uint16]
+    clusterId*: uint16
+    shardIds*: seq[uint16]
 
 func clusterId*(rs: RelayShards): uint16 =
```

Comment:
remove the `clusterId` and `shardIds` if they are no longer needed.

Reply:
sure: c6a446b4502f6ec6a727bae99e3d8d87b7190844